plugwise/data.py

```diff
@@ -225,26 +226,26 @@ def _get_adam_data(self, entity: GwEntityData, data: GwEntityData) -> None:
         """Helper-function for _get_entity_data().
 
         Determine Adam heating-status for on-off heating via valves,
-        available regulations_modes and thermostat control_states.
+        available regulations_modes and thermostat control_states,
+        and add missing cooling_enabled when required.
         """
-        if self.smile(ADAM):
+        if entity["dev_class"] == "heater_central":
```

Comment:
Maybe we should 'match/case' this, but at least just return before the match against gateway (as that wouldn't be processed e.g. there is nothing more to do once heater central is done)?